Rollbar/RollbarConfiguration.h

```diff
@@ -22,15 +22,19 @@
 - (void)save;
 
 - (void)setPersonId:(NSString*)personId username:(NSString*)username email:(NSString*)email;
+- (void)setPayloadModificationBlock:(void (^)(NSDictionary*))payloadModificationBlock;
+- (void)setRequestId:(NSString*)requestId;
 
 - (NSDictionary *)customData;
 
 @property (atomic, copy) NSString *accessToken;
 @property (atomic, copy) NSString *environment;
 @property (atomic, copy) NSString *endpoint;
 @property (atomic, copy) NSString *crashLevel;
-@property (readonly, atomic, copy) NSString *personId;
-@property (readonly, atomic, copy) NSString *personUsername;
-@property (readonly, atomic, copy) NSString *personEmail;
+@property (readonly, nonatomic, copy) NSString *personId;
+@property (readonly, nonatomic, copy) NSString *personUsername;
+@property (readonly, nonatomic, copy) NSString *personEmail;
+@property (nonatomic, copy) void (^payloadModification)(NSDictionary *payload);
```

Comment:
I like the use of a block for doing transformations. We might want to think about having this more abstractly as part of a sequence of payload modifications, but that might be overkill now that I am thinking about it. I just don't want to end up in the scenario where we have one special case thing that transforms the payload that stands apart from every other one that we have. But maybe this is fundamentally different. Looks good for now.